src/core/thread/mle.cpp

```diff
@@ -2139,10 +2139,6 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
             mNetif.GetPendingDataset().Set(pendingTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
-    else
```

Comment:
Ok, got it. So we will not take 0 as a special timestamp value, I will update the PR, thanks.